doc/source/examples/extended_examples/gui/executable.rst

```diff
@@ -0,0 +1,119 @@
+:orphan:
```

Comment:
```suggestion
```

This example shouldn't be an orphan because it is included in the toctree in the `index.rst` file of `extended_examples`.